src/common/camera.h

```diff
@@ -39,8 +44,13 @@ class camera {
         // the normalized image-based coordinates of the pixel. Image left is s=0, image right
         // is s=1, image top is t=0, image bottom is t=1.
 
+        // Jitter the sample location of the pixel.
```

Comment:
Jitter is not quite correct here.

Jitter is the colloquial term given for the placement of samples within a _stratified sampling_.

That's not exactly what we're doing. We're just randomly sampling within a pixel. It's not stratified within a pixel, so it's not jitter.

Replace with something like
// Randomly place the pixel sample location

Reply:
Excellent point. Will do.